spec/models/spree/gateway/shopify_gateway_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'spec_helper'
+
+module Spree
+  RSpec.describe Gateway::ShopifyGateway do
+    let(:transaction_id) { '0xDEADBEEF' }
+    let(:pos_order_id) { '0xBAADF00D' }
+    let(:refund) { double('refund', pos_order_id: pos_order_id) }
+    let(:refund_reason) { double('refund_reason', name: 'Product not working') }
+    let(:gateway_options) { { originator: refund } }
+
+    let(:provider_class) { ActiveMerchant::Billing::ShopifyGateway }
+    let(:provider_instance) { double('provider', refund: true, void: true) }
+
+    subject(:gateway) do
+      described_class.create!(name: "Shopify")
+    end
+
+    before do
+      gateway.preferences = { api_key: ENV['SHOPIFY_API_KEY'],
+                              password: ENV['SHOPIFY_PASSWORD'],
+                              shop_name: ENV['SHOPIFY_SHOP_NAME'] }
+
+      allow(provider_class).to receive(:new).and_return(provider_instance)
+    end
+
+    describe "#payment_profiles_supported?" do
+      it { is_expected.to_not be_payment_profiles_supported }
```

Comment:
Would this read more 'eloquent' using a string to describe the `it` over the shorthand?